pytr/timeline.py

```diff
@@ -198,10 +200,12 @@ def finish_timeline_details(self, dl):
                 indent=2,
             )
 
-        export_transactions(
-            dl.output_path / "all_events.json",
-            dl.output_path / "account_transactions.csv",
-            sort=dl.sort_export,
-        )
+        with (dl.output_path / "account_transactions.csv").open("w") as f:
```

Comment:
```suggestion
        with (dl.output_path / "account_transactions.csv").open("w", encoding="utf8") as f:
```
Should be utf8 here, too.